OverPassLayer.js

```diff
@@ -229,6 +244,11 @@ L.OverPassLayer = L.FeatureGroup.extend({
           var queryWithMapCoordinates = this.options.query.replace(/(BBOX)/g, bbox.toOverpassBBoxString());
           var url =  this.options.endpoint + "interpreter?data=[out:json];" + queryWithMapCoordinates;
 
+          if (beforeRequest) {
```

Comment:
If you put `beforeRequest()` at the beginning of `onMoveEnd` so the `beforeRequest()` can be called without making any request. For example:

 1. zoom in up to the minZoom -> some requests are sent
 2. zoom in one more time -> no requests are sent because they are skipped in line 235 (`continue;`)

That's why we can't put the `beforeRequest()` at the beginning of `onMoveEnd`